magento/magento2
# MC-13537: added default Country Id to checkout Config.

### Description (*)
The reason of that bug is: the _DefaultCountryId_ sets from the config path tax/defaults/country and not from general/country/default of Magento 2 configuration. That's why when you're go to the checkout and that's your first time, session wasn't saved in browser, table rates are wrong. 


### Fixed Issues (if relevant)

1. magento/magento2#<[13537](https://github.com/magento/magento2/issues/13537)>: Magento 2.2.0 One page checkout initial postage calculation ignores "default country" using table rate shipping.

### Manual testing scenarios (*)

1. Set Default country to "United Kingdom".
2. Setup table rate shipping.
3. Add item to cart.
4. Proceed to one page checkout.
5. Default country is set automatically to "United Kingdom" as expected.
6. The initial shipping price displayed is that of the USA but should be the UK.
7. Deselect then reselect "United Kingdom" the shipping price refreshes correctly.
